backend/experiment/views.py

```diff
@@ -87,20 +87,22 @@ def get_experiment_collection(request, slug):
         return Http404
     request.session[COLLECTION_KEY] = slug
     participant = get_participant(request)
-    active_groups = ExperimentCollectionGroup.objects.filter(
-        series=collection.id, finished=False).order_by('order')
-    if not active_groups:
-        # TO DO: We should return a debrief here
-        return JsonResponse({})
```

Comment:
This part is not represented in the refactor, does that mean there should not be any debrief here (anymore)?

Reply:
No, the debrief is now moved to the `rhythm_battery_final`. We may eventually give an option to add an introductory / outro explainer to a `ExperimentCollection`, but I thought that for now, this was a good enough fix.